src/app/core/providers/effects/list-effects.service.ts

```diff
@@ -90,11 +93,24 @@ export class ListEffectsService {
 
     /**
      * make a comma seperated list of langues. Put the passed language in front
-     * @param language 
+     * @param language
      */
     private languageWithFallbacks(language: string): string {
         return this.config.CONTENT_LANGUAGES
             .sort((a, b) => a === language ? -1 : 1)
             .join(',');
     }
+
+    /**
+     *
+     */
+    public deleteNode(node: MeshNode, recursive: boolean): void {
+        this.api.project.deleteNode({ project: node.project.name, nodeUuid: node.uuid, recursive })
+            .take(1)
```

Comment:
take(1) is unnecessary when calling Api methods, since all the http methods are single-value observables (i.e. the stream completes after the first value)